cime_config/acme/machines/config_batch.xml

```diff
@@ -51,6 +51,21 @@
      </submit_args>
    </batch_system>
 
+  <batch_system type="cobalt_theta" version="x.y">
```

Comment:
Cobalt on Mira uses `runjob <runjob_args> : <executable>`.
Cobalt on Theta uses `aprun <aprun_args> <executable>`.
The colon-separator is added elsewhere, outside of XML, and causes a job submission error.

There is also now support for `#COBALT ` directives similar to `#PBS ` directives and having a new batch system may be useful